portfolio/src/main/java/com/google/sps/data/Message.java

```diff
@@ -0,0 +1,29 @@
+package com.google.sps.data;
+
+public class Message {
+    public String comment;
+    private long author;
+    private long timestamp;
+    private String email;
+
+    // Email should be optional
+    public Message(String c, long a, long t, String e){
+        comment=c; 
+        author=a; 
+        timestamp=t; 
+        email=e;
+    }
+
+    public Message(String c, long a, long t){
+        comment=c; 
+        author=a; 
+        timestamp=t; 
+        email=null;
+    }
+
+    public String getComment(){return comment;}
```

Comment:
Typically at Google, we would add spacing:

public String getComment() {
  return comment;
}